tooling/build/scripts/generate-sitemap.js

```diff
@@ -90,6 +90,16 @@ const getSiteMapEntry = async (fullPath, relativePath, name) => {
     tags: schemaData.page.tags,
   }
 
+  if (schemaData.layout === "collection") {
```

Comment:
issue: I think we also need to do this for `index.ts` inside `publishing` for it to take effect for Studio sites

Reply:
thanks! updated in https://github.com/opengovsg/isomer/pull/1250/commits/2c6bd3cefd3a7633d3bb77e367f015c60bb34ebe